src/main/java/uk/gov/hmcts/reform/sscs/service/DwpAddressLookupService.java

```diff
@@ -60,44 +60,56 @@ public Address lookupDwpAddress(SscsCaseData caseData) {
         }
     }
 
-    DwpAddress lookup(String benefitType, String dwpIssuingOffice) {
-        Optional<DwpAddress> dwpAddressOptional = getDwpAddress(benefitType, dwpIssuingOffice);
+    OfficeAddress lookup(String benefitType, String dwpIssuingOffice) {
+        OfficeAddress dwpAddress = getDwpAddress(benefitType, dwpIssuingOffice);
 
-        if (!dwpAddressOptional.isPresent()) {
-            throw new DwpAddressLookupException(String.format("could not find dwp address for benefitType %s and dwpIssuingOffice %s",
+        if (dwpAddress == null) {
+            throw new DwpAddressLookupException(String.format("could not find dwp officeAddress for benefitType %s and dwpIssuingOffice %s",
                     benefitType, dwpIssuingOffice));
         }
-        return dwpAddressOptional.get();
+        return dwpAddress;
     }
 
-    private Address buildAddress(DwpAddress dwpAddress) {
+    private Address buildAddress(OfficeAddress dwpAddress) {
         return Address.builder()
-            .line1(dwpAddress.getLine1().orElse(null))
-            .town(dwpAddress.getLine2().orElse(null))
-            .county(dwpAddress.getLine3().orElse(null))
-            .postcode(dwpAddress.getPostCode().orElse(null))
+            .line1(dwpAddress.getLine1())
+            .town(dwpAddress.getLine2())
+            .county(dwpAddress.getLine3())
+            .postcode(dwpAddress.getPostCode())
             .build();
     }
 
-    public Optional<DwpAddress> getDwpAddress(String benefitType, String dwpIssuingOffice) {
-        log.info("looking up address for benefitType {} and dwpIssuingOffice {}", benefitType, dwpIssuingOffice);
+    public OfficeAddress getDwpAddress(String benefitType, String dwpIssuingOffice) {
+        log.info("looking up officeAddress for benefitType {} and dwpIssuingOffice {}", benefitType, dwpIssuingOffice);
 
         if (StringUtils.equalsIgnoreCase(dwpIssuingOffice, TEST_HMCTS_ADDRESS)) {
-            return Optional.of(TEST_ADDRESS);
+            return dwpMappings.getTestHmctsAddress().getAddress();
         }
         if (StringUtils.equalsIgnoreCase(PIP, benefitType)) {
             String dwpIssuingOfficeStripped = dwpIssuingOffice.replaceAll("\\D+","");
-            return Optional.ofNullable(PIP_LOOKUP.get(StringUtils.stripToEmpty(dwpIssuingOfficeStripped)));
+            return getOfficeAddressByOffice(dwpIssuingOfficeStripped);
         } else if (StringUtils.equalsIgnoreCase(ESA, benefitType)) {
-            return Optional.ofNullable(ESA_LOOKUP.get(StringUtils.stripToEmpty(dwpIssuingOffice)));
+            return getOfficeAddressByOffice(dwpIssuingOffice);
         }
-        return Optional.empty();
+        return null;
```

Comment:
your reversing the pattern to use an anti-pattern

Reply:
Gson couldn't serialise the POJO objects from the JSON when Optionals were used, hence the reason these were removed.